config/environments/production.rb

```diff
@@ -32,16 +25,20 @@
   # Do not fallback to assets pipeline if a precompiled asset is missed.
   config.assets.compile = false
 
-  # Asset digests allow you to set far-future HTTP expiration dates on all assets,
-  # yet still be able to expire them through the digest params.
-  config.assets.digest = true
-
```

Comment:
Is not part of the default config – I plan to test this on staging to check if we still have digests?